src/types.rs

```diff
@@ -273,11 +273,12 @@ pub struct SyncStatus {
     pub is_syncing: bool,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
do we even need the `Debug` impl?

Reply:
only to print the result to the console for testing.

Reply:
ok fine